Makefile

```diff
@@ -18,6 +18,7 @@ ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 # Base path used to install.
 DESTDIR ?= /usr/local
+KUBE_TEST_REPO_LIST ?=
```

Comment:
Can we rename this to something else that doesn't reference Kubernetes? Might be a tad confusing :)